### ISSUE TITLE
[5.x] Allow users to specify kubectl version on operator host

### ISSUE THREAD

**Issue by matthewmcdaniel** [oracle-terraform-modules/terraform-oci-oke#831]
Hi team. It appears that users cannot specify the version of kubectl that is installed on the operator host, and defaults to the latest release in OLCNE 1.5 (which appears to be v1.24.15). I think this is important because it will violate the K8s version skew policy for clusters above v1.25.

**Comment by robo-cap** [comment]
I think we need to move to OLCNE 1.7: https://public-yum.oracle.com/repo/OracleLinux/OL8/olcne17/x86_64/index.html

Changes required: 
```
  ol8_olcne17 = {
          name     = "Oracle Linux Cloud Native Environment 1.7 ($basearch)"
          baseurl  = "https://yum$ociregion.$ocidomain/repo/OracleLinux/OL8/olcne17/$basearch/"
          gpgkey   = "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-oracle"
          gpgcheck = true
          enabled  = true
        }
```
in here: https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/5.x/modules/operator/cloudinit.tf#L39-L45

```
# yum info kubectl

Available Packages
Name         : kubectl
Version      : 1.26.6
Release      : 1.el8
Architecture : x86_64
Size         : 11 M
Source       : kubernetes-1.26.6-1.el8.src.rpm
Repository   : ol8_olcne17
Summary      : Command-line utility for interacting with a Kubernetes cluster.
URL          : https://kubernetes.io
License      : ASL 2.0
Description  : Command-line utility for interacting with a Kubernetes cluster.
```

**Comment by hyder** [comment]
Resolved in #836

**Comment by Spechal** [comment]
Partially fixed ...
